src/components/visualizations/utils/common.ts

```diff
@@ -49,6 +49,10 @@ export function getAttributeElementIdFromAttributeElementUri(attributeElementUri
     return attributeElementUri.slice(attributeElementUri.lastIndexOf(match) + match.length);
 }
 
+export function isRotationInRange(rotation: number, min: number = 75, max: number = 105) {
```

Comment:
when function is called `isRotationInRange`, I would definitely not have defaults here and would always pass them.